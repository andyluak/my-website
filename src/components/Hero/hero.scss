@media screen and (max-width: 768px ) {
	.hero {
		text-align: center;
		h1 {
			padding: 1em 0em;
		}
		p {
			padding-bottom: 1em;
		}

		.hero_image {
			display:flex;
			justify-content: center;
		}
	}
}

@media screen and (min-width: 768px) {
	.hero {
		display:flex;
		flex-direction: row-reverse;
		justify-content: flex-start;
		align-items: center;
		padding: 1em 6em;
		justify-content: space-between;
		text-align: right;

		h1 {
			padding: 1em;
		}

		p {
			padding: 2em;
		}
	}
}