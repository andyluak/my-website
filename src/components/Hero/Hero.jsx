import './hero.scss';
import { ReactComponent as Avatar } from '../../assets/images/avatar.svg';

import React from 'react';

const Hero = () => {
	return (
		<div className="hero">
			<div className="hero_text">
				<h1>I build stuff</h1>
			</div>

			<div className="hero_image">
				<Avatar />
			</div>
		</div>
	)
}

export default Hero;