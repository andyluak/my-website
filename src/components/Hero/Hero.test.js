import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Hero from './Hero';

describe( 'Hero component tests', () => {
	it( 'excepct to render Hero component', () => {
		expect(shallow(<Hero />)).toMatchSnapshot();
	})
})

