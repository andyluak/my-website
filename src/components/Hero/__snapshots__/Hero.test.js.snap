// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero component tests excepct to render Hero component 1`] = `
<div
  className="hero"
>
  <div
    className="hero_text"
  >
    <h1>
      I build stuff
    </h1>
  </div>
  <div
    className="hero_image"
  >
    <ForwardRef(SvgAvatar) />
  </div>
</div>
`;
